packages/react-composites/src/composites/ChatComposite/adapter/AzureCommunicationChatAdapter.ts

```diff
@@ -476,6 +509,18 @@ const convertEventToChatMessage = (event: ChatMessageReceivedEvent): ChatMessage
   };
 };
 
+const convertEventToDeletedChatMessage = (event: ChatMessageDeletedEvent): DeletedChatMessage => {
+  return {
+    id: event.id,
+    version: event.version,
+    type: convertEventType(event.type),
+    sender: event.sender,
+    senderDisplayName: event.senderDisplayName,
+    sequenceId: '',
+    createdOn: new Date(event.createdOn)
```

Comment:
Do we need to set "deletedOn" here too?

Reply:
100%. And this means we should remove the `deletedOn: Date` in the MessageDeletedListener as well as its redundant.
I should do the same for editedOn

Reply:
Thanks updated! This also meant I could simplify a bunch of pieces and remove the DeletedChatMessage interface as it was redundant also